cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala

```diff
@@ -71,10 +71,10 @@ class FulltextIndexTest extends DocumentingTest {
         #Using this feature, it is possible to work around the slow Lucene writes from the performance critical commit process, thus removing the main bottlenecks for Neo4j write performance.""".stripMargin('#'))
     p("""At first sight, the construction of full-text indexes can seem similar to regular indexes.
         #However there are some things that are interesting to note:
-        #In contrast to <<administration-indexes-search-performance, b-tree indexes>>, a full-text index can be:""".stripMargin('#'))
+        #In contrast to the <<administration-indexes-search-performance, other indexes>>, a full-text index can be:""".stripMargin('#'))
```

Comment:
I'm not sure about 'other' here. Perhaps 'standard' or 'regular'. I'm not sure what's best.

Reply:
Not sure, lets see what the docs team says?

Reply:
I'd say `regular` or simply drop the article. Prefer the latter.

Reply:
so something like this?
```suggestion
        #In contrast to <<administration-indexes-search-performance, other indexes>>, a full-text index can be:""".stripMargin('#'))
```

Reply:
Exactly, I did leave an in-line suggestion too, looks exactly like this :) .

Reply:
the fact that I removed my comment when I noticed your suggestion makes this comment thread interesting now :P